treeherder/etl/runnable_jobs.py

```diff
@@ -207,7 +207,15 @@ def _buildbot_runnable_jobs(project):
     return ret
 
 
+def _ensure_buildbot_data_is_loaded():
+    if not RunnableJob.objects.all():
+        RunnableJobsProcess().run()
+
+
 def list_runnable_jobs(project, decision_task_id=None):
+    # Bug 1329685 - This guarantees that calling list_runnable_jobs always returns the correct data
+    # even when running Treeherder for the first time
+    _ensure_buildbot_data_is_loaded()
```

Comment:
I'm slightly uneasy about adding this here - it will be triggered as part of API requests, which just feels wrong.

There are lots of things that aren't loaded by default in a Vagrant instance (eg bugscache), I think we need to decide a strategy for them all. eg one of:
* document additional ingestion, but no code changes
* add additional commands like ingest_push to allow easy population of an empty Vagrant instance
* ...

I'd like to make the change to load_initial_data.py in the meantime at least.